pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -67,7 +71,11 @@ t_ndarray   array_create(int nd, int *shape, enum e_types type);
 void        _array_fill_int(int c, t_ndarray arr);
 void        _array_fill_float(float c, t_ndarray arr);
 void        _array_fill_double(double c, t_ndarray arr);
-void        _array_fill_cdouble(complex double c, t_ndarray arr);
+void        _array_fill_bool(bool c, t_ndarray arr);
+void        _array_fill_sint(short int c, t_ndarray arr);
+void        _array_fill_long(long c, t_ndarray arr);
+void        _array_fill_cfloat(float complex c, t_ndarray arr);
+void        _array_fill_cdouble(double complex c, t_ndarray arr);

```

Comment:
Where are the definitions of these functions?

Reply:
they are already in the `ndarray.c` file.